packages/interface/src/features/home/hooks/useFetchFaucet.ts

```diff
@@ -0,0 +1,15 @@
+import { ZKEdDSAEventTicketPCDClaim } from "@pcd/zk-eddsa-event-ticket-pcd";
+
+import { api } from "~/utils/api";
+
+import type { PCD } from "@pcd/pcd-types";
+import type { UseTRPCQueryResult } from "@trpc/react-query/shared";
+import type { Groth16Proof } from "snarkjs";
+import type { IFetchFaucetReturn } from "~/utils/types";
+
+export function useClaimFunds(
+  address: string,
+  pcd: PCD<ZKEdDSAEventTicketPCDClaim, Groth16Proof>,
+): UseTRPCQueryResult<IFetchFaucetReturn, unknown> {
```

Comment:
Is it possible to not use unknown here?

Reply:
I think it's hard, since the error type is hard to know beforehand, same as other api useQuery hook